cypress/fixtures/customReportData.json

```diff
@@ -0,0 +1,23 @@
+{
+  "customFooter": {
+    "PageFooter": "Custom footer for main page",
+    "Account": "<div><p><b>Custom footer for Account pages</b></p></div>",
+    "Device": "<div><p><b>Custom footer for Device pages</b></p></div>"
+  },
+  "customMetadata": {
+    "Account": {
+      "Setup": {
+        "PreReq1": "SetupAccount1",
```

Comment:
We might not need PreReq 1, 2 and 3. A single key under moduleName would've sufficed, if the value can contain line breaks.
Example:
`
"customMetadata": {
    "Account": "SetUpAccount1</br>SetupAccount2......",
    "Device": "SetupDevice1</br>SetupDevice2...."
}
`
These do not have to be <li> list items, a <p> paragraph would suffice as mentioned in ticket description.